### ISSUE TITLE
sqlx db connection with ssh

### ISSUE THREAD

**Issue by jaykam** [jmoiron/sqlx#727]
Hi @jmoiron 

Is it possible to make a DB connection with ssh using the SQLX library?

Thanks

**Comment by xNevo** [comment]
No, for this you need to setup a SSH tunnel first on your client

**Comment by jaykam** [comment]
@xNevo Yeah, got it thanks!

**Comment by gurza** [comment]
@jaykam look at https://gist.github.com/vinzenz/d8e6834d9e25bbd422c14326f357cce0 sample
